NameAppender.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+/*
+ * This is a simple golang console application that asks for your name as input
+ * and appends your name to the end of the README.md file in this repository.
+ * Author: trey.watford@acstechnologies.com
+ * Date: 10/17/16
+ */
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"runtime"
+	"strings"
+)
+
+//use newline character as line delimiter
+const inputDelimiter = '\n'
+
+func main() {
+	//prompt for user input
+	fmt.Print("Please enter your name: ")
+
+	//create input reader to read line from stdin
+	reader := bufio.NewReader(os.Stdin)
+
+	//read from stdin ending in newline
+	in, err := reader.ReadString(inputDelimiter)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	//remove CRLF in windows strings and LF in unix strings
+	if runtime.GOOS == "windows" {
```

Comment:
you could do

```
res = strings.Replace(res, "\r", "", -1) //this will only be useful under windows
res = strings.Replace(res, "\n", "", -1)
```

It eliminates the if statement because there is nothing to replace. I use it in my logging package.

https://github.com/dixonwille/wlog/blob/master/basic.go#L85-#L86